packages/runtime/src/utils/load.ts

```diff
@@ -137,6 +161,11 @@ export async function getRemoteEntry({
         entry,
         remoteEntryExports,
       });
+    } else if (type === 'system') {
```

Comment:
Should we add type system as well ?

Reply:
It's already present: https://github.com/module-federation/core/blob/09b792d31259eeb4d33f6e4975d7c333aab5e6d5/packages/sdk/src/types/stats.ts#L21